src/pages/settings/Profile/CustomStatus/CustomClearAfterPage.js

```diff
@@ -0,0 +1,88 @@
+import lodashGet from 'lodash/get';
+import moment from 'moment';
+import React, {useCallback} from 'react';
+import {withOnyx} from 'react-native-onyx';
+import FormProvider from '@components/Form/FormProvider';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import NewDatePicker from '@components/NewDatePicker';
+import ScreenWrapper from '@components/ScreenWrapper';
+import withLocalize, {withLocalizePropTypes} from '@components/withLocalize';
+import * as User from '@libs/actions/User';
+import compose from '@libs/compose';
+import DateUtils from '@libs/DateUtils';
+import Navigation from '@libs/Navigation/Navigation';
+import * as ValidationUtils from '@libs/ValidationUtils';
+import styles from '@styles/styles';
+import ONYXKEYS from '@src/ONYXKEYS';
+import ROUTES from '@src/ROUTES';
+
+const propTypes = {
+    ...withLocalizePropTypes,
+};
+
+function CustomClearAfterPage({translate, customStatus}) {
+    const customClearAfter = lodashGet(customStatus, 'clearAfter', '');
+
+    const onSubmit = (v) => {
+        User.updateDraftCustomStatus({clearAfter: DateUtils.combineDateAndTime(customClearAfter, v.dob)});
+        Navigation.goBack(ROUTES.SETTINGS_STATUS_CLEAR_AFTER);
+    };
+
+    const validate = useCallback((values) => {
+        const requiredFields = ['dob'];
```

Comment:
What does DOB mean

Reply:
The name was copied from here https://github.com/margelo/expensify-app-fork/blob/91b8be1e7f91e3a6df4f2ee88cd4705d982d6f42/src/pages/settings/Profile/PersonalDetails/DateOfBirthPage.js#L84
i'll rename it